roles/lib_openshift/library/oc_adm_ca_server_cert.py

```diff
@@ -958,13 +962,13 @@ def _process(self, template_name, create=False, params=None, template_data=None)
 
         return self.openshift_cmd(['create', '-f', fname])
 
-    def _get(self, resource, rname=None, selector=None):
+    def _get(self, resource, name=None, selector=None):
         '''return a resource by name '''
         cmd = ['get', resource]
-        if selector:
-            cmd.append('--selector=%s' % selector)
-        elif rname:
-            cmd.append(rname)
+        if selector is not None:
```

Comment:
@ashcrow, good question.  I just tried a query with both given and it is an error when both are supplied.

```
oc get template mysql --selector=template=mysql-ephemeral-template --all-namespaces
error: name cannot be provided when a selector is specified
```

I will update the module params to be mutually-exclusive.